.gitmodules

```diff
@@ -23,7 +23,3 @@
 	url = https://github.com/google/benchmark
 	branch = master
 
-[submodule "third_party/opentelemetry-proto"]
```

Comment:
Is CMake build affected by this?

Reply:
Will double check

Reply:
Ok, because I don't know how to download an archive with cmake, I made bazel to use the released version, and point the submodule to the same tag.

Reply:
Can't we not just update the submodule version, and let cmake and bazel use that ( as they are doing right now ).

Reply:
https://github.com/open-telemetry/opentelemetry-cpp/issues/461

Reply:
@lalitb - we have to update submodule version. Latest looks good to me.

Reply:
the submodule was updated to point to the same tag release commit